reference/debugging.md

```diff
@@ -0,0 +1,300 @@
+# Debugging
+
+If you’re experiencing issues with Ambassador and cannot diagnose the issue through the "Diagnostics" tab from the [Edge Policy Console](../../about/edge-policy-console), this document covers various approaches and advanced use cases for debugging Ambassador issues.
+
+First, create an example configuration for debugging demonstrations that are available throughout this document.
+
+Then, complete the following debugging options:
+
+* Check Ambassador Status
+* Review Ambassador Logs
+* Examine Pod and Container Contents
+* Customize the Envoy Configuration
+
+## Example Configuration
+
+To see a demonstration of how you can debug your Ambassador instance, follow the instructions to create an example mapping configuration.
+
+Note: The following assumes that you deployed Ambassador and the following services from the [quick start installation guide](../../user-guide/install) to a Kubernetes cluster.
+
+**In the command line:**
+
+1. Create a cluster in GKE with RBAC support enabled, and your user account configured correctly. Then run the following:
+
+    ```shell
+    $ gcloud container clusters create ambassador-demo --preemptible
+    $ kubectl create clusterrolebinding cluster-admin-binding-new \
+    --clusterrole cluster-admin --user <your_user_name>
+    ```
+
+2. Deploy the latest version of Ambassador:
+
+    ```shell
+    $ kubectl apply -f https://getambassador.io/yaml/ambassador/ambassador-rbac.yaml
+    ```
+
+3. Next, create an Ambassador Service and deploy a basic `httpbin` Ambassador Mapping by saving the following YAML to a file named `ambassador-services.yaml`
+
+    ```yaml
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+    name: ambassador
+    spec:
+    type: LoadBalancer
+    ports:
+    - port: 80
+    selector:
+    service: ambassador
+    ---
+    apiVersion: getambassador.io/v1
+    kind:  Mapping
+    metadata:
+    name:  httpbin
+    spec:
+    prefix: /httpbin/
+    service: httpbin.org
+    host_rewrite: httpbin.org
+    ```
+
+4. Apply this into your cluster with the following command:
+
+    ```shell
+    $ kubectl apply -f ambassador-services.yaml
+    ```
+
+You should now be able to utilize this example mapping for debugging demonstrations and purposes.
+
+## Check Ambassador Status
+
+First, check to see if the [Diagnostics](/reference/diagnostics) service is reachable with the command: `kubectl port-forward <ambassador_pod_name> 8877`
```

Comment:
Use `../diagnostics` to avoid linking to the GA docs.
```suggestion
First, check to see if the [Diagnostics](../diagnostics) service is reachable with the command: `kubectl port-forward <ambassador_pod_name> 8877`
```

Reply:
But also, the `8877` thing is kinda deprecated with Edge Stack. I'm not sure what this step should be replaced with. (@kflynn?)